### PR TITLE
docs(global header): Pattern for review

### PR DESCRIPTION
**Pull request by designertyler**
Adds the global header pattern

Preview here: https://carbon-website-git-fork-designertyler-global-header-1212.carbon-design-system.now.sh/patterns/global-header

### CODE REVIEW

File: src/pages/patterns/global-header/index.mdx [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+
+Users who rely on screen reader technology to navigate areas a site also rely on screen readers to navigate the content of a page. In the same way users can visually scan for the larger and bolder type of a header, users of assistive technologies also must understand the hierarchy of the page's content to efficiently navigate a page's content.
+
+To ensure all users interpret the structure of your content in the same way, the visual representation of a heading should match the underlying ranking of the header tag. Tutorials on how to achieve this can be found in the WCAG page structure [tutorials on headings](https://www.w3.org/WAI/tutorials/page-structure/headings/).
+
+### Keyboard navigation
+
+#### Skip to main content
+
+Some users may use a keyboard to navigate your site. Starting focus in the main nav lets them quickly navigate to other areas in your UI, but could block them from the main content if there is a large number of nav items to tab through first.
```
Comment (janchild) [inline src/pages/patterns/global-header/index.mdx]:
```suggestion
Some users may use a keyboard to navigate your site. Starting focus in the main navigation lets them quickly navigate to other areas in your UI, but could block them from the main content if there is a large number of navigation items to tab through first.
```

File: src/pages/patterns/global-header/index.mdx [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+
+[Success Criterion 2.4.1 (Bypass Blocks)](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G1) suggest bypassing these blocks by providing "Skip to main" link at the start of you navigation's focusable controls. This lets users easily skip the navigation region and being interacting with the page's main content area.
```
Comment (janchild) [inline src/pages/patterns/global-header/index.mdx]:
```suggestion
[Success Criterion 2.4.1 (Bypass Blocks)](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G1) suggest bypassing these blocks by providing "Skip to main" link at the start of you navigation's focusable controls. This lets users easily skip the navigation region and begin interacting with the page's main content area.
```

File: src/pages/patterns/global-header/index.mdx [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+Examples of common landmark regions include: navigation, main, form, banner, and search. If there are multiple navigation landmark regions give each a unique label.
+
+<Row>
+
+<Column  colLg={8} colMd={8} colSm={6} >
+
+![hint text](images/landmark_zones.png)
+
+<Caption>
+  1. Navigation landmark. <Br />
```
Comment (janchild) [inline src/pages/patterns/global-header/index.mdx]:
```suggestion
  1. Navigation landmark <Br />
```

File: src/pages/patterns/global-header/index.mdx [hunk 4]
```diff
@@ -0,0 +1,299 @@
...
+  2. Search landmark. <Br />
```
Comment (janchild) [inline src/pages/patterns/global-header/index.mdx]:
```suggestion
  2. Search landmark <Br />
```

File: src/pages/patterns/global-header/index.mdx [hunk 5]
```diff
@@ -0,0 +1,299 @@
...
+  3. Main landmark. <Br />
```
Comment (janchild) [inline src/pages/patterns/global-header/index.mdx]:
```suggestion
  3. Main landmark <Br />
```

File: src/pages/patterns/global-header/index.mdx [hunk 6]
```diff
@@ -0,0 +1,299 @@
...
+
+Navigating between landmark regions helps users who cannot see the visual grouping of your navigation. This grouping can be important to understanding the organization of the structure of the content and making it clear what users can do and where they can go in your UI.
+
+<Row>
+
+<Column  colLg={8} colMd={8} colSm={6} >
+
+![hint text](images/voiceover.png)
+
+<Caption>Screenshot of a rotor in action.</Caption>
```
Comment (janchild) [inline src/pages/patterns/global-header/index.mdx]:
```suggestion
<Caption>Screenshot of a rotor in action</Caption>
```

File: src/pages/patterns/global-header/index.mdx [hunk 7]
```diff
@@ -0,0 +1,299 @@
...
+- [Breadcrumb](https://www.carbondesignsystem.com/components/breadcrumb/code/)
+
+## References
+
+- David R. Danielson, [_Transitional Volatility in Web Navigation_](https://www.researchgate.net/publication/240859594_Transitional_volatility_in_web_navigation) (2003)
+- Susan Farrell, [_Utility Navigation: What It Is and How to Design It_](https://www.nngroup.com/articles/utility-navigation/) (2015)
+- IBM Design, [_Accessibility Handbook_](http://accessibility-handbook.mybluemix.net/design/a11y-handbook/) (2019)
+- James Kalbach, [_Designing Web Navigation_](https://www.oreilly.com/library/view/designing-web-navigation/9780596528102/ch04.html) (2007)
+- WebAIM, [_"Skip Navigation" Links_](https://webaim.org/techniques/skipnav/) (2013)
+- Web Content Accessibility Guidelines, [_WCAG_](https://www.w3.org/WAI/standards-guidelines/wcag/) (2018)
```
Comment (janchild) [inline src/pages/patterns/global-header/index.mdx]:
```suggestion
- [Web Content Accessibility Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/) (W3C, 2018)
```

File: src/pages/patterns/global-header/index.mdx [hunk 8]
```diff
@@ -0,0 +1,299 @@
...
+</Row>
+
+## Configurations
+
+The UI shell is configurable to let a product or platform choose which shell components and configurations match their user and information requirements. Simple products have the flexibility of using a header or side panel and more complex products require a combination of UI shell components to accommodate the depth of their navigation.
+
+![Example the header and left nav being used for the same IA.](images/configurations.png)
+
+<Caption>
+  A simple product navigation can be achieved with either a header or side panel
```
Comment (janchild) [inline src/pages/patterns/global-header/index.mdx]: Is it possible to include real IBM product examples here?

Comment (designertyler) [reply src/pages/patterns/global-header/index.mdx]: The only real examples that exist today are the IBM design and the Carbon sites. I had those in there earlier but was told to take them out. I could mock up some fake ones for now if that's what we want.

Comment (shinytoyrobots) [reply src/pages/patterns/global-header/index.mdx]: Possible tweak to this text;

"Simpler navigation patterns have the flexibility of using either a header or side panel. More complex navigation needs can be accommodated with a combination of UI shell components."

Main intent is to talk about complexity of navigation versus complexity of product.

**Pull request merged**